code/modules/mob/living/carbon/xenomorph/castes/hivelord/abilities_hivelord.dm

```diff
@@ -107,7 +107,7 @@
 	if(!.)
 		return FALSE
 	var/turf/T = get_turf(owner)
-	if(locate(/obj/structure/tunnel) in T)
+	if(locate(/obj/structure/xeno/tunnel) in T)
```

Comment:
why are you not using check_alien_construction ?

Reply:
this is how it always was, but I'll consolidate

Reply:
actually this is deliberate; it's so you can hide the tunnels